src/main/java/longtimenosee/model/event/Event.java

```diff
@@ -0,0 +1,125 @@
+package longtimenosee.model.event;
+
+import static longtimenosee.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Comparator;
+import java.util.List;
+
+/**
+ * Represents an Event in the address book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+
+public class Event {
+
+    public static final Comparator<Event> EVENT_COMPARATOR = new Comparator<Event>() {
+        @Override
+        public int compare(Event e1, Event e2) {
+            int x = e1.getDate().getDate().compareTo(e2.getDate().getDate());
```

Comment:
Perhaps you can use a more descriptive variable name here

Reply:
thank you for pointing that out!